darts/tests/test_global_forecasting_models.py

```diff
@@ -234,15 +233,12 @@ def test_fit_with_fit_epochs(self, rmtree_patch, train_patch, save_patch):
 
                 model.fit(multiple_ts, epochs=epochs)
 
-                rmtree_patch.assert_called()
```

Comment:
One last thing. This tested whether `shutil.rmtree` was called, right? Is there a way we could still have a test case that covers this behavior when `reset_model` is called? I hope this makes sense, if not sorry, I'm not too familiar with this `patch` functionality.

Reply:
I'm just a bit paranoid about removing test cases as darts is getting bigger and bigger

Reply:
yes, but with this specific testcase rmtree should not be called - is that what you want to check? Just to make sure, that was mine test for the previous implementation and for me it didn't make sense to keep it there as the test is about something else.

Reply:
Makes sense!